app/models/orders_package.rb

```diff
@@ -84,6 +86,11 @@ def update_designation(order_id_to_update)
     update(order_id: order_id_to_update, updated_by: User.current_user)
   end
 
+  def redesignate(new_order_id)
```

Comment:
is it worth wrapping this in a transaction so that the database is rolled back if there is an error partway through? https://api.rubyonrails.org/classes/ActiveRecord/Transactions/ClassMethods.html

I'm assuming here that `redesignate` is the main model API entry point for this action.

Reply:
oh that's great ! the api certainly lacks atomicity, this is perfect !